examples/00-pyunits/basic_usage.py

```diff
@@ -269,3 +269,20 @@
 
 meters_per_second.value  # >>> 3.4137599999999995
 meters_per_second.units  # >>> "m s^-1"
+
+
+###############################################################################
+# Using preferred units
+# ~~~~~~~~~~~~~~~~~~~~~
+# Specify a list of units that quantities will automatically convert to.
+
+Quantity.preferred_units(units=["J"])
```

Comment:
@ansaboutin 

This is a good design choice.

Does it work equivalently if I write the following?
```
Quantity.preferred_units(units=UnitRegistry().J)
```

Reply:
`Quantity.preferred_units(units=[UnitRegistry().J])` Would work. I have it set up to take in lists currently. Should I also get it to work with single units? @seanpearsonuk